fastlane_bot/tests/test_064_TestMultiAllMode.py

```diff
@@ -224,7 +224,7 @@ def test_test_expected_output():
     
     assert len(r) >= 25, f"[NBTest64 TestMultiPairwiseAll Mode] Expected at least 25 arbs, found {len(r)}"
     assert multi_carbon_count > 0, f"[NBTest64 TestMultiPairwiseAll Mode] Not finding arbs with multiple Carbon curves."
-    assert carbon_wrong_direction_count == 0, f"[NBTest64 TestMultiPairwiseAll Mode] Expected all Carbon curves to have the same tkn in and tkn out. Mixing is currently not supported."
+    # assert carbon_wrong_direction_count == 0, f"[NBTest64 TestMultiPairwiseAll Mode] Expected all Carbon curves to have the same tkn in and tkn out. Mixing is currently not supported."
```

Comment:
That's not an ideal way to handle it.
Instead of masking out a test, you should make it work.

Reply:
There is no way to modify this test such that it works.
The underlying principal is that pairs in the "wrong direction" can be supported for intra Carbon, as is the focus of this PR, but also potentially for all pairs. The latter requires more research and, if true, a redesign and optimization of the pairs mode - outside of the scope of this PR.
So TLDR to add support for intra carbon we need to pass "wrong direction" carbon pairs and so this breaks the test